docker/OpenVINO_DLDT/Dockerfile

```diff
@@ -34,40 +41,46 @@ RUN apt-get update && \
     apt-get install -y -qq --no-install-recommends ${DEPENDENCIES} && \
     rm -rf /var/lib/apt/lists/*
 
-#install openvino
-RUN wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB && \
-    sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
-RUN echo "deb https://apt.repos.intel.com/openvino/2022 bionic main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list && \
-    sudo apt update && echo y | sudo apt install openvino-2022.1.0 openvino-opencv
+# Install OpenVINO
+RUN sudo mkdir /opt/intel && \
+    curl -L https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.2/linux/l_openvino_toolkit_ubuntu20_2022.2.0.7713.af16ea1d79a_x86_64.tgz --output openvino_2022.2.0.7713.tgz && \
+    tar -xf openvino_2022.2.0.7713.tgz && \
+    sudo mv l_openvino_toolkit_ubuntu20_2022.2.0.7713.af16ea1d79a_x86_64 /opt/intel/openvino_2022.2.0.7713 && \
+    sudo ln -s /opt/intel/openvino_2022.2.0.7713 /opt/intel/openvino_2022
+
+RUN tar -xf openvino_2022.2.0.7713.tgz && \ 
+    ls -laR l_openvino_toolkit_ubuntu20_2022.2.0.7713.af16ea1d79a_x86_64/install_dependencies
+
+RUN ls /opt/intel/openvino_2022/
 
-#install openvino dependencies
-ENV INSTALL_DIR /opt/intel/openvino_2022
+# Install OpenVINO dependencies
+ENV INSTALL_DIR /opt/intel/openvino_2022/
 WORKDIR $INSTALL_DIR/install_dependencies
-RUN echo y | ./install_openvino_dependencies.sh
+RUN ls -la /opt/intel/openvino_2022/install_dependencies && echo y | ./install_openvino_dependencies.sh
 RUN cat /root/.bashrc > tmp && echo 'source $INSTALL_DIR/setupvars.sh' > /root/.bashrc
 
-#configurations for GPU, MYRIAD
+# Configure for GPU, MYRIAD
 RUN echo y | ./install_NEO_OCL_driver.sh
 RUN usermod -a -G video,users root
 
-#openvino-dev
+# Install openvino-dev
 WORKDIR /tmp/
-RUN pip3 install --upgrade pip && \
-    pip3 install openvino-dev==2022.1
+RUN pip3 install --upgrade pip
+RUN pip3 install openvino-dev==2022.2
 
-#accuracy-check
-RUN git clone --recursive https://github.com/openvinotoolkit/open_model_zoo.git --branch 2022.1.0 --single-branch
+# Test Accuracy Checker
+RUN git clone --recursive https://github.com/openvinotoolkit/open_model_zoo.git --branch 2022.2.0 --single-branch --depth 1
 WORKDIR /tmp/open_model_zoo/tools/accuracy_checker
 RUN wget https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz
 RUN tar xvf cifar-10-python.tar.gz -C sample
-RUN pip3 uninstall opencv-python -y && pip3 install "opencv-python-headless<4.7"
+RUN pip3 uninstall opencv-python -y && pip3 install "opencv-python-headless"
 RUN /bin/bash -c "accuracy_check -c sample/sample_config.yml -m data/test_models -s sample"
 
-#DLB
+# Download DLI source code
 WORKDIR /tmp/
-RUN git clone https://github.com/itlab-vision/dl-benchmark.git && \
+RUN git clone https://github.com/itlab-vision/dl-benchmark.git --depth 1 && \
     pip3 install docker PyYAML
 
-#DLB dataset
+# Download dataset
 ARG DATASET_DOWNLOAD_LINK=<Link to dataset project>
```

Comment:
Cannot find documentation (neither in README nor in Wiki) about how to resolve the placeholder "< Link to dataset project >"; without replacing the placeholder the following "RUN git clone $DATASET_DOWNLOAD_LINK" will fail, and the container cannot be built, right?
Can you make it optional or provide a default value, i.e. pointing to a "default/dummy git project"?

Reply:
Hello, as a repository, you can now submit any repository, even an empty one. In the future, we will add some public repository.

Reply:
Yes, sure. At the moment I hd just commented-out the "git clone" command.
Could you add a note to the documentation (README, Wiki) to inform the users to replace/update the argument manually before trying to build the container.

Reply:
@brmarkus we made this parameter optional and add note about this in README in docker directory